src/components/LimitedAccessBanner/LimitedAccessBanner.js

```diff
@@ -25,19 +26,29 @@ const LimitedAccessBanner = props => {
   const user = ensureCurrentUser(currentUser);
 
   const showBanner =
-    user.id &&
-    isAuthenticated &&
-    authScopes &&
-    authScopes.length === 1 &&
-    authScopes[0] === 'user:limited' &&
-    !disabledPages.includes(currentPage);
+    user.id && isAuthenticated && isLoggedInAs && !disabledPages.includes(currentPage);
 
   const { firstName, lastName } = user.attributes.profile;
 
+  const limitedRights = authScopes.indexOf('user:limited') >= 0;
+  const fullRights = authScopes.indexOf('user') >= 0;
```

Comment:
@bladealslayer 

> as it will match on string that contains user:limited

No, I don't think it will match. `authScopes` is an array and `indexOf` is returning the index in the array

Reply:
Ah, 🤦, true. I thought this is working on strings.